src/Orangebeard.Client/Entities/Log.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orangebeard.Client.Entities
```

Comment:
Do we need separate classes for logs and attachments? Is a log with an attachment not just also a log, but with an extra, optional field (the file) filled? This feels a bit redundant

Reply:
This is how it is done in the Java client. I try to make the .NET client as similar to the Java client as I reasonably can.